addons/hr_expense/tests/test_expenses_access_rights.py

```diff
@@ -27,6 +27,31 @@ def test_expense_access_rights(self):
                 'price_unit': 1,
             })
 
+        expense = self.env['hr.expense'].with_user(self.expense_user_employee).create({
+            'name': 'expense_1',
+            'date': '2016-01-01',
+            'product_id': self.product_a.id,
+            'price_unit': 10.0,
```

Comment:
```suggestion
            'quantity': 10.0,
```
I know it's not what's tested here but let's test the flows as they really are